Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageContainerView.swift

```diff
@@ -193,7 +193,12 @@ struct MessageContainerView<Factory: ViewFactory>: View {
     }
     
     private var spacerWidth: CGFloat {
-        (width ?? 0) / 4
+        let width = width ?? 0
+        if isIPad {
+            return 2 * width / 3
+        } else {
+            return width / 4
+        }
```

Comment:
A little confusing as to which width is which (tongue twister alarm 🚨)...
Maybe call the local constant of `width` something like `proposedViewWidth` or something that makes more sense?

Reply:
Yes, makes sense, I'll update that.